### PR TITLE
Updating approvers list

### PR DESCRIPTION
**Pull request by karthikv2k**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/fairing/72)

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -1,7 +1,7 @@
 approvers:
   - wbuchwalter
   - r2d4
-reviewers:
   - jlewi
```
Comment (jlewi) [inline OWNERS]: Alphabetize please.

**Comment by jlewi** [comment]
What's the motivation for adding me as an approver?

Should we also remove @wbuchwalter I don't think he has been very active of late with this repo.

**Comment by wbuchwalter** [comment]
I haven't been very active recently indeed. Hope to be more active in a few months once I clear some things off my plate, but in the mean time you can remove me from approvers as you see fit.

**Comment by karthikv2k** [comment]
> What's the motivation for adding me as an approver?
> 
> Should we also remove @wbuchwalter I don't think he has been very active of late with this repo.

Just wanted have more than one active owner who can help out in approving the PRs. Synced with Matt and will add @rpasricha and myself to the owners list. Also will remove @wbuchwalter from approvers for time being.

**Comment by r2d4** [comment]
Ah, looks like you'll need to rebase since @rpasricha added himself as a reviewer in a previous commit

https://github.com/kubeflow/fairing/pull/70

**Pull request merged**